sys-fs/cryfs/cryfs-0.10.3.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+inherit cmake flag-o-matic linux-info python-any-r1
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/cryfs/cryfs"
+else
+	SRC_URI="https://github.com/cryfs/cryfs/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+	S="${WORKDIR}"
+fi
+
+DESCRIPTION="Encrypted FUSE filesystem that conceals metadata"
+HOMEPAGE="https://www.cryfs.org/"
+
+LICENSE="LGPL-3 MIT"
+SLOT="0"
+IUSE="custom-optimization debug test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/boost-1.65.1:=
+	>=dev-libs/crypto++-8.2.0:=
+	net-misc/curl:=
+	>=sys-fs/fuse-2.8.6:0
+	dev-libs/openssl:0=
+"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	test? ( dev-cpp/gtest )
+"
+
+PATCHES=(
+	# TODO upstream:
+	"${FILESDIR}/0.10.2-unbundle-libs.patch"
+	"${FILESDIR}/0.10.2-install-targets.patch"
```

Comment:
You forgot to test after changing ;)

You're missing a ${PN} here. It doesn't apply right now:
```
>>> Preparing source in /var/tmp/portage/sys-fs/cryfs-0.10.3/work ...
 * Applying 0.10.2-unbundle-libs.patch ...
/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/environment: line 1611: /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch: No such file or directory
/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/environment: line 1614: /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch: No such file or directory     [ !! ]
 * ERROR: sys-fs/cryfs-0.10.3::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch
 *
 * Call stack:
 *               ebuild.sh, line  125:  Called src_prepare
 *             environment, line 3516:  Called cmake_src_prepare
 *             environment, line 1528:  Called default_src_prepare
 *      phase-functions.sh, line  920:  Called __eapi6_src_prepare
 *             environment, line  237:  Called eapply '/var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch' '/var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-install-targets.patch'
 *             environment, line 1679:  Called _eapply_patch '/var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch'
 *             environment, line 1617:  Called __helpers_die 'patch -p1  failed with /var/tmp/portage/sys-fs/cryfs-0.10.3/files/0.10.2-unbundle-libs.patch'
 *   isolated-functions.sh, line  112:  Called die
 * The specific snippet of code:
 *              die "$@"
 *
 * If you need support, post the output of `emerge --info '=sys-fs/cryfs-0.10.3::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=sys-fs/cryfs-0.10.3::gentoo'`.
 * The complete build log is located at '/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/sys-fs/cryfs-0.10.3/temp/environment'.
 * Working directory: '/var/tmp/portage/sys-fs/cryfs-0.10.3/work'
 * S: '/var/tmp/portage/sys-fs/cryfs-0.10.3/work'
```

I'll fix